docs/mesos-containerizer.md

```diff
@@ -66,3 +66,33 @@ Mesos supports the following built-in isolators.
 - [volume/secret](secrets.md#file-based-secrets)
 - [windows/cpu](isolators/windows.md#cpu-limits)
 - [windows/mem](isolators/windows.md#memory-limits)
+
+
+## Systemd Integration
+
+<table class="table table-striped">
```

Comment:
I am not sure if we need a table here, maybe just putting the following statement here should be good enough?

> To prevent systemd from manipulating cgroups managed by the agent, it's recommended to add `Delegate` under `Service`, for example:
> ```
> [Service]
> Delegate=true
> ```

Reply:
I have no special feelings for the table. :-) If you prefer without, I will remove it. Then I can also cleanup the CR in the agent.md by the way.

Reply:
Yeah, let's remove the table and cleanup the CR in the agent.md : )

Reply:
I'm back after holiday :-)

Reply:
Welcome back, I just added a few minor comments :-)

Reply:
Thanks :-)